### ISSUE TITLE
Settings modal section titles need more space

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/mail#9229]
### Steps to reproduce

1. Open the app
2. Configure an account
3. Open the app settings in the bottom left

### Expected behavior

Nice visuals

### Actual behavior

| Header | Header |
|--------|--------|
| ![Bildschirmfoto vom 2024-01-08 18-15-22](https://github.com/nextcloud/mail/assets/1374172/9dc59a40-1086-43c1-9101-8de643c1ebed) | ![Bildschirmfoto vom 2024-01-08 18-15-27](https://github.com/nextcloud/mail/assets/1374172/1e321e31-c86f-491c-b106-4a7fdccd96f4) | 

### Mail app version

3.5

### PHP engine version

None

### Web server

None

### Database

None

**Comment by ChristophWurst** [comment]
@nimishavijay we use https://nextcloud-vue-components.netlify.app/#/Components/App%20containers/NcAppSettings?id=ncappsettingsdialog. Should that component have a wider column for the section names or be responsive depending on text length?

**Comment by ChristophWurst** [comment]
Hovering ellipsized headings does *not* show the full text. This might also be an a11y issue. cc @JuliaKirschenheuter

**Comment by jancborchardt** [comment]
Yes it would probably be good to make the navigation a bit wider to also account for other languages.

However it seems mail settings also has an additional issue, and all the relevant items fall into that: 1 section for 1 setting, which ends up in way too many sections:
- Activate body search (first off, really weird naming … :D more understandable would be "Search in mail body text")
- Data collection consent
- Reply text position
- Also "Writing mode"
- Generally: Any setting that is only a single checkbox or an either-or setting.

All of these do not need their own sections, but can be put in one section called e.g. "General" which would be towards the top, probably directly below the aliases.

**Comment by madbob** [comment]
The settings modal has been completely redesigned in #12031
